pkg/controller/masteruserrecord/sync.go

```diff
@@ -185,8 +185,14 @@ func (s *Synchronizer) alignReadiness() (bool, error) {
 	if condition.IsNotTrue(s.record.Status.Conditions, toolchainv1alpha1.MasterUserRecordUserProvisionedNotificationCreated) {
 		notification := &toolchainv1alpha1.Notification{
 			ObjectMeta: v1.ObjectMeta{
-				Name:      s.record.Name + "-provisioned",
+				GenerateName:      s.record.Name + "-" + toolchainv1alpha1.NotificationTypeProvisioned + "-",
```

Comment:
```suggestion
				GenerateName:      fmt.Stringf("%s-%s-", s.record.Name, toolchainv1alpha1.NotificationTypeProvisioned,
```

Reply:
Fixed as part of: 21c323901080723156e76ab6563614f16880baf3. I used Sprintf here - I think Stringf may be a typo.

Reply:
Yep. Thanks :)